content/applications/studio/automated_actions.rst

```diff
@@ -1,249 +1,308 @@
+To do so, select or create an :guilabel:`Email Template` or an an :guilabel:`SMS Template`, then, in
+the :guilabel:`Send Email As` or :guilabel:`Send SMS As` field, choose how you want to send the
+email or text message.
 
 .. _studio/automated-actions/action/add-followers:
 
-Add Followers
+Add Followers and Remove Followers
+----------------------------------
+
+Use these actions to (un)subscribe existing contacts to the record.
+
+Create Record
 -------------
 
-The action is used to subscribe existing contacts to the record.
+This action is used to create a new record on any model.
 
-.. _studio/automated-actions/action/next-activity:
+.. note::
+   Selecting a :guilabel:`Record to Create` is only required if you want to target a model other
+   than the one you are on.
 
-Create Next Activity
---------------------
+Specify a :guilabel:`Name` for the record, then, optionally, select a field in the
+:guilabel:`Link Field` field to link the record that triggered the creation of the new record.
 
-The action is used to schedule a new activity linked to the record. Use the :guilabel:`Activity` tab
-to set it up as usual, but instead of the :guilabel:`Assigned to` field, select an
-:guilabel:`Activity User Type`. Select :guilabel:`Specific User` and add the user under
-:guilabel:`Responsible` if the activity should always be assigned to the same user. To dynamically
-target a user linked to the record, select :guilabel:`Generic User From Record` instead and change
-the :guilabel:`User field name` if necessary.
+.. tip::
+   You can create another automation rule with :ref:`studio/automated-actions/action/update-record`
+   actions to update the fields of the new record if necessary. For example, you can use a
+   :guilabel:`Create Record` action to create a new project task and then assign it to a specific
+   user using an :guilabel:`Update Record` action.
 
-.. example::
-   After a lead is turned into an opportunity, you want your automated action to set up a call for
-   the user responsible for the lead. To do so, set the :guilabel:`Activity` to :guilabel:`Call`
-   and set the :guilabel:`Activity User Type` to :guilabel:`Generic User From Record`.
+.. _studio/automated-actions/action/python-code:
 
-   .. image:: automated_actions/next-activity-example.png
-      :align: center
-      :alt: Example of a Create Next Activity action
+Execute Code
+------------
 
-.. _studio/automated-actions/action/send-sms:
+This action is used to execute Python code. The available variables are described in the
+:guilabel:`Python Code` tab, which is also used to write your code, or in the :guilabel:`Help` tab.
 
-Send SMS Text Message
----------------------
+Send Webhook Notification
+-------------------------
 
-The action is used to send an SMS to a contact linked to the record. To do so, select or create an
-:guilabel:`SMS Template`.
+This action allows to send a POST request with the values of the :guilabel:`Fields` to the URL
```

Comment:
stupid question, but what's a POST request? Is it an abbreviation for something? If so, what for? I'd also suggest :abbr:

Reply:
my 2 cents:
> Do you think i need to add some sort of explanation

no

> if you're venturing into webhooks, you should know what you're doing

i agree 💯